server/sonar-server/src/main/java/org/sonar/server/ruby/PlatformRubyBridge.java

```diff
@@ -42,44 +42,29 @@ public PlatformRubyBridge(RackBridge rackBridge) {
 
   @Override
   public RubyDatabaseMigration databaseMigration() {
-    final CallDatabaseVersionUpgrade callDatabaseVersionUpgrade = parseMethodScriptToInterface(
+    CallDatabaseVersionUpgrade callDatabaseVersionUpgrade = parseMethodScriptToInterface(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "callDatabaseVersionUpgrade". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1854)